api/openapi-spec/openapi.yaml

```diff
@@ -315,7 +315,7 @@ components:
       required:
         - id
         - key
-    Attribute:
+    OptimizelyAttribute:
```

Comment:
It looks ok but want to confirm that theres is no implications of these renaming to existing agent users.

Reply:
Since we have only updated the model names and haven't removed/renamed any old `key/value` pair in the optimizelyConfig, there shouldn't be any implication i believe. What do you think? @msohailhussain